sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -79,3 +116,27 @@ def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, TokenCre
         )
         self._deserialize = _get_deserialize(self._api_version)
         self._generated_models = self._client.models(self._api_version)
+
+    @distributed_trace
+    def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs) -> HttpResponse:
+        """Runs a network request using the client's existing pipeline.
+
+        The request URL can be relative to the vault URL. The service API version used for the request is the same as
+        the client's unless otherwise specified. This method does not raise if the response is an error; to raise an
+        exception, call `raise_for_status()` on the returned response object. For more information about how to send
+        custom requests with this method, see https://aka.ms/azsdk/dpcodegen/python/send_request.
+
+        :param request: The network request you want to make.
+        :type request: ~azure.core.rest.HttpRequest
+
+        :keyword bool stream: Whether the response payload will be streamed. Defaults to False.
+
+        :return: The response of your network call. Does not do error handling on your response.
+        :rtype: ~azure.core.rest.HttpResponse
+        """
+        request_copy = _format_api_version(request, self._api_version)
+        path_format_arguments = {
+            "vaultBaseUrl": _SERIALIZER.url("vault_base_url", self._endpoint, "str", skip_quote=True),
```

Comment:
It's probably not `vaultBaseUrl` :)

Reply:
For reference, KV uses `vaultBaseUrl` because this is used as the base endpoint in the generated client:
https://github.com/Azure/azure-sdk-for-python/blob/a193fbc54bdd8145a4b0643939af19a8ea2abb32/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_generated/v7_4/_key_vault_client.py#L31-L33
For FR, the base URL is instead `endpoint` -- there's also a version attached to this endpoint, so you'd need to do some testing to see how this affects targeting API versions:
https://github.com/Azure/azure-sdk-for-python/blob/3269c622a1bc0f5cc8fc757ba88a36848491d130/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2_1/_form_recognizer_client.py#L43-L45